gradle/libs.versions.toml

```diff
@@ -279,6 +280,7 @@ google-auth-credentials = { group = "com.google.auth", name = "google-auth-libra
 
 aliyun-credentials-sdk = { group='com.aliyun', name='credentials-java', version.ref='aliyun-credentials' }
 flinkjdbc = {group='org.apache.flink',name='flink-connector-jdbc', version.ref='flinkjdbc'}
+jcasbin = {group='org.casbin',name='jcasbin',version.ref="jcasbin"}
```

Comment:
Can you update the style like the above line, to add necessary whitespace before and after.